docs/api-reference/effects/glfx-filters.md

```diff
@@ -0,0 +1,350 @@
+# glfx Filters
+
+
+Image effects packaged as reusable shader modules in `@luma.gl/effects` based on the [glfx library](http://evanw.github.io/glfx.js/).
```

Comment:
Please structure the documents properly so that we don't break links in the follow up releases.

In documentation tree:

Effects
|- Overview
|- glfx shader modules
|- (future effects in this module)

In "Overview":

The `@luma.gl/effects` module offers modular, reusable image effects.

```
## Installation

    npm install @luma.gl/effects

## Attribution

...
```

In "glfx shader modules":

Everything in "Filters"

Reply:
There is no "filter" concept in luma.gl. Let's call these "shader modules" for consistency.

Reply:
good suggestion, done.